service/ses/api_op_GetIdentityVerificationAttributes.go

```diff
@@ -341,29 +340,18 @@ func (w *IdentityExistsWaiter) WaitForOutput(ctx context.Context, params *GetIde
 func identityExistsStateRetryable(ctx context.Context, input *GetIdentityVerificationAttributesInput, output *GetIdentityVerificationAttributesOutput, err error) (bool, error) {
 
 	if err == nil {
-		pathValue, err := jmespath.Search("VerificationAttributes.*.VerificationStatus", output)
-		if err != nil {
-			return false, fmt.Errorf("error evaluating waiter state: %w", err)
+		v1 := output.VerificationAttributes
```

Comment:
[observation] This wildcard took a 2nd reading. The model is 

```
VerificationAttributes map[string]types.IdentityVerificationAttributes
```

So iterating through the values gets us the same result as the wildcard